### ISSUE TITLE
Missing classifier: sentiment_classifier_big.obj

### ISSUE THREAD

**Issue by rtanglao** [turtlelovesshoes/SUMOTwitterInsights#11]
Hi @turtlelovesshoes @benmiroglio 

**tl;dr Could you please upload sentiment_classifier_big.obj to github?**

Filing this issue for the dataviz and insights firm Analyse and Tal out of Denmark, in particular one of their folks whose name is Håvard,  we are collaborating with for SUMO insights (they are looking at all of our prior art and code around insights including this one!):

Håvard wrote in slack:

<blockquote>
Hey @roland, in the Twitter Sentiment repo you sent, we're missing the classifier itself which I guess you guys have created based on some training data. Do you think it would be possible for us to get? (edited) 

As far as I can tell from this script (https://github.com/turtlelovesshoes/SUMOTwitterInsights/blob/master/sentiment/train.py) it's a pickle file called *sentiment_classifier_big.obj*
</blockquote>

Cheers and happy new year!

...Roland

**Comment by benmiroglio** [comment]
Hey Roland,

The object is too big for Github, but you can download the file here:

https://drive.google.com/open?id=1mqyy89KjgmshbMD7nA8aUDQscXdyBBHF

**Comment by rtanglao** [comment]
> 
> 
> Hey Roland,
> 
> The object is too big for Github, but you can download the file here:
> 
> https://drive.google.com/open?id=1mqyy89KjgmshbMD7nA8aUDQscXdyBBHF

awesome thanks Ben!

**Comment by Taarnborg** [comment]
Hi Roland,

This is just a heads up, that  I've  made a permission request to access til classifier. 
I work in Analyse & Tal together with Håvard.

**Comment by rtanglao** [comment]
> 
> 
> Hi Roland,
> 
> This is just a heads up, that I've made a permission request to access til classifier.
> I work in Analyse & Tal together with Håvard.

@Taarnborg did you get want you want, if not how can i help?

**Comment by rtanglao** [comment]
i'll handle @Taarnborg 's issue separately. Everything else has been resolved, so closing this issue.

**Comment by Taarnborg** [comment]
I got the classifier, thx!
